source/bson-binary-vector/tests/README.md

```diff
@@ -55,6 +55,40 @@ MUST assert that the input float array is the same after encoding and decoding.
 - if the canonical_bson field is present, raise an exception when attempting to deserialize it into the corresponding
     numeric values, as the field contains corrupted data.
 
+## Prose Tests
+
+### Treatment of non-zero ignored bits
+
+Drivers SHOULD test this behavior, whether an error is thrown or bits are preserved, according to their design and
+version. For drivers that haven't been completed, raise an exception. For those that have, update to this behavior
+according to semantic versioning rules.
+
+A vector of length 1 with non-zero ignored bits provides a good example to use.
+
+#### 1. Encoding
```

Comment:
Can you make these a bit more prose-y, in particular for Decoding I'm not sure what python is returning from find_one that then needs to be passed to "as_vector". Just some 1,2,3 steps of what to create and what to assert would be great

Reply:
Ok. I haven't seen many prose tests. Will do!